protocol-specs/package.json

```
@@ -23,6 +23,7 @@
 		"@liskhq/lisk-transactions": "2.3.1",
 		"@liskhq/lisk-validator": "0.2.1",
 		"lodash": "4.17.15",
+		"protobufjs": "^6.9.0",
```

Please use a fixed version. 

```suggestion
		"protobufjs": "6.9.0",
```

---

Fixed